packages/tools/pluggable-widgets-tools/bin/mx-scripts.js

```diff
@@ -70,6 +70,14 @@ function getRealCommand(cmd, toolsRoot) {
             return `jest --projects ${join(toolsRoot, "test-config/jest.config.js")}`;
         case "test:unit:native":
             return `jest --projects ${join(toolsRoot, "test-config/jest.native.config.js")}`;
+        case "test:e2e:native:local:ios":
```

Comment:
I would not add this to pluggable widget toos if we are not shipping detox on it. Please move the scripts and tasks outside of it.